### ISSUE TITLE
Problem with unencrypted fetchmail

### ISSUE THREAD

**Issue by Woipi** [Mailu/Mailu#3670]
## Environment & Version

- Version: `2024.06`

## Replication Steps

1. Setup a Fetchmail task
2. Deselect TLS encryption

## Observed behaviour

Fetchmail tries to upgrade to an TLS connection:

LOG: "fetchmail: xxx.xxx.xxx.xxx: upgrade to TLS failed."
LOG: "fetchmail: Socket or TLS error on xxx"
LOG: "fetchmail: socket error while fetching from xxx"

This renders the checkbox useless.

## Expected behaviour

Fetchmail not utilizing TLS when deselecting the box.

## Solution

Change fetchmail.py from:

`options += " ssl" if fetch["tls"] else ""`

to:

`options += " ssl" if fetch["tls"] else " sslproto \'\'"`

This causes fetchmail to not automatically upgrade to a TLS connection when the checkbox is deselected. This behaviour is for example necessary for the email-oauth2-proxy.

**Comment by nextgens** [comment]
Right, the behavior of fetchmail has changed in v6.4.0 (https://www.fetchmail.info/fetchmail-man.html)
